.gitignore

```diff
@@ -10,6 +10,7 @@ goblint
 goblint.byte
 goblint.json
 goblint.domaintest
+goblint-http
```

Comment:
Isn't it named `goblint-http.exe` now? Could also make the rule `goblint-http*` I guess.